networkx/utils/backends.py

```diff
@@ -732,7 +753,7 @@ def _call_if_any_backends_installed(self, /, *args, backend=None, **kwargs):
                     ):
                         # Log that we are falling back to networkx
                         _logger.debug(
-                            "Backend '%s' can't run `%s'. %s",
+                            "Backend '%s' can't run '%s'. %s",
```

Comment:
I surprised you changed <code>`</code> to <code>'</code> so many places. This was first introduced in #7300, and I followed the style. Anyway, there are a few places in exception messages that still use the former.